app/src/main/java/com/example/mytestproject/ui/searchCity/SearchCityFragment.kt

```diff
@@ -0,0 +1,126 @@
+package com.example.mytestproject.ui.searchCity
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.SearchView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.mytestproject.App
+import com.example.mytestproject.R
+import com.example.mytestproject.util.MINIMUM_SYMBOLS
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Flowable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.fragment_search_city.*
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+class SearchCityFragment : Fragment() {
```

Comment:
In task 15 there is a requirement about displaying 10 last cities.
You probably missed it?

Reply:
Yes, I forgot.
Fixec